src/excmds.ts

```diff
@@ -4380,6 +4572,18 @@ export function jumble() {
         body.currentNode.textContent = jumble_helper(t)
     }
 }
+
+/**
+ * Run a command in a tab with an identifier specified by destination or in background if destination is "background".
```

Comment:
Should probably specify that destination is an internal mozilla tab id and not the 1-based tab index we assign them